src/data_readers/data_reader_jag_conduit.cpp

```diff
@@ -428,6 +429,10 @@ void data_reader_jag_conduit::check_input_keys() {
 
 #ifndef _JAG_OFFLINE_TOOL_MODE_
 void data_reader_jag_conduit::load() {
+  if(m_gan_labelling) m_num_labels=2;
+  std::cout << "JAG load GAN m_gan_labelling : label_value "
```

Comment:
Do you want all processors to write to cout -- or is the printing just for use during development?

Reply:
Good point. I think I will add if (is_master()) {},